tasks/sshd.yml

```diff
@@ -56,4 +56,5 @@
   selinux_permissive:
     name: sshd_t
     permissive: true
-  when: ansible_selinux|bool
+  when: ( ansible_selinux is defined and ( (ansible_selinux is mapping and ansible_selinux.status == 'enabled') or ( ansible_selinux|bool ) )  )
```

Comment:
I'm confused. Is still possible for `ansible_selinux` to be a `bool` even after https://github.com/ansible/ansible/pull/31065/files was merged?

Some report the following (a little bit neater) should work as well:

`when: ansible_selinux is defined and ansible_selinux != False and ansible_selinux.status == 'enabled'`

I suspect this can work because it relies on a "fail early" approach in python condition evaluation. When ansible_selinux == False, the last condition is not even checked (which will raise an error). Thoughts?

Reply:
I used the one suggested, but it fail on molecule lint:

https://travis-ci.org/theoapp/ansible-role-theo-agent/jobs/653445907#L527

Reply:
maybe `when: ansible_selinux is defined and ansible_selinux and ansible_selinux.status == 'enabled'` will make the linter happy

Reply:
let's try